db/migrate/20231024200452_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :likes_counter
+      t.integer :comments_counter
+      t.integer :author_id, index: true
+
+      t.timestamps
+    end
+
+    add_foreign_key :posts, :users, column: :author_id
+  end
```

Comment:
- [ ] Great job so far 👍🏻. However, the foreign key column is not correct, we are always us the column type `references` (also available as `belongs_to`) to create an association between tables. For more documentation, you may visit this link [Creating associations using references](https://guides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references). 
- It could be something like this:
```  ruby
create_table :posts do |t|
    t.string :title
    ...
    t.references :author, foreign_key: { to_table: 'users' } 
  end
```
- This will create author_id and a corresponding index.

Reply:
Hi @BenMukebo Thank you for the review.
Changes done: updated the association in the post table using the type references for author column.